### ISSUE TITLE
Embeds export as gray squares

### ISSUE THREAD

**Issue by claudelesclave** [tldraw/tldraw#1872]
### What happened?

exporting a tldraw whiteboard results in link or specifically link previews being grey squares without pictures or text of the link preview

![bug](https://github.com/tldraw/tldraw/assets/37047824/83cd3255-939c-4f89-8e1d-513b1abb7d9b)

### How can we reproduce the bug?

Insert a link, and export as svg, png (both transparent and not) also copy as png or svg results to the same

### What browsers are you seeing the problem on?

Chrome, Safari

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by MitjaBezensek** [comment]
Yeah, it would be great to support this. Currently we use iFrames to render embeds, so the 3rd party does the actually rendering. I guess for this to work we would have to browser render the embed and replace them in the export.
